Cargo.lock

```diff
@@ -550,7 +550,7 @@ dependencies = [
  "proc-macro-error-attr",
  "proc-macro2",
  "quote",
- "syn",
+ "syn 1.0.109",
```

Comment:
Hrmm, we should figure out which crate is inducing this dependency and get everything fully upgraded

Reply:
Ah it's `pearlite-syn`

Reply:
Well... If our dependencies use a different version of Syn, is this a problem?